src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt

```diff
@@ -49,13 +49,22 @@ class VarselService(
                 narmesteLeder = narmesteLeder,
                 varseltype = varselType,
             )
-        } else {
-            val altinnMelding = createAltinnMelding(virksomhetsbrevId, virksomhetsnummer, virksomhetsPdf, varselType, arbeidstakerPersonIdent, arbeidstakernavn)
-            altinnClient.sendToVirksomhet(
-                altinnMelding = altinnMelding,
-            )
         }
 
+        val altinnMelding = createAltinnMelding(
```

Comment:
Foreslår at kallet til altinnClient går først, og så narmesteLederVarselService.sendVarsel etterpå. 

Jeg tror altinnClient er det som feiler oftest, så best å kalle den først slik at vi ikke gjør kall til narmesteLederVarselService.sendVarsel  for ting som feiler etterpå.